### DISCUSSION TITLE
Mod sorting & Holding

### DISCUSSION THREAD

**Discussion by Jadan1213** [modrinth/.github#196] [Feature requests]
It would be wonderful if the ability to add an option to sort mods by available updates. This way you don't have to sift through each page looking for updated versions in scenarios where you may want to update mods one at a time or look to see what updates are available.

Another feature I think would be great is the ability to HOLD a certain mod version for when using the "Update All" button. This would avoid a scenario where a specific version is required and shouldn't be updated due to incompatibility, or a scenario where the update to a mod may be in Alpha or Beta stage and you don't want to update it (or maybe the ability to skip auto updating of Alpha and Beta mod updates).

**Comment by Its3rr0rsWRLD** [comment]
To update all at once, select all of the mods (top left selection box) then press update, good idea though! Lets see what they say 😊

**Comment by Jadan1213** [comment]
@Its3rr0rsWRLD I already know how to update all mods. My request is pertaining to sorting by update (so you can check the updates easily before applying them) and holding mods from updates (in order to keep a specific version). There's already an update all button and you don't need to select any of the mods to use it.

**Reply by Its3rr0rsWRLD** [reply]
I agree, would love to contribute, and although they probably have it open source, I can't find any repo for the application, *I literally haven't looked im in a car right now so yeah lol 😭💀*
